modules/mono/editor/GodotTools/GodotTools/GodotSharpEditor.cs

```diff
@@ -515,6 +582,20 @@ public override void _EnablePlugin()
                 ["hint_string"] = settingsHintStr
             });
 
+            _editorSettings.AddPropertyInfo(new Godot.Collections.Dictionary
+            {
+                ["type"] = (int)Variant.Type.String,
+                ["name"] = Settings.CustomExecPath,
+                ["hint"] = (int)PropertyHint.File,
+            });
+
+
+            _editorSettings.AddPropertyInfo(new Godot.Collections.Dictionary
+            {
+                ["type"] = (int)Variant.Type.String,
+                ["name"] = Settings.CustomExecPathArgs,
+            });
+
```

Comment:
```suggestion
            _editorSettings.SetInitialValue(Settings.CustomExecPathArgs, "{file}", false);

```
To make the default behavior clear and match the global external editor.